etl/command.py

```diff
@@ -22,6 +22,11 @@
 # if the number of open files allowed is less than this, increase it
 LIMIT_NOFILE = 4096
 
+# DAG file to use by default.
+# Use paths.DAG_ARCHIVE_FILE to load the complete dag, with active and archive steps.
+# Otherwise use paths.DAG_FILE to load only active steps, ignoring archive ones.
+DEFAULT_DAG_FILE = paths.DAG_ARCHIVE_FILE
```

Comment:
I'd be eager to do the leap of faith and make un-archived DAG the default one (unless you want to test it first in production with archived+ DAG and then do a second PR for un-archived DAG).

Reply:
Yes, I'm happy to do the leap of faith too :-D. But for now I'll stick to what we decided with Lars last time we talked about it. It's easy enough to change it in a second PR.